go/framework/bertybridge/bridge_protocol.go

```diff
@@ -144,8 +147,10 @@ func newProtocolBridge(logger *zap.Logger, config *ProtocolConfig) (*Protocol, e
 			}
 
 			var bopts = ipfsutil.CoreAPIConfig{
-				APIAddrs:  DefaultAPIAddrs,
-				APIConfig: APIConfig,
+				SwarmAddrs:        []string{defaultMCBind},
```

Comment:
Can you update the berty config to have `DefaultSwarmAddrs` instead of `DefaultMCBind`, then add all default addrs to it (`tcp` + `quic` + `DefaultMCBind`)
You will need to update all addrs passed by the JS as well.
This will be more readable with less side effect